### PR TITLE
Add a JupyterHub sudospawner recipe

### PR DESCRIPTION
**Pull request by parente**

### CODE REVIEW

**Comment by parente** [comment]
@minrk I listed you as a co-maintainer on the recipe. Just let me know if you'd rather I didn't.

**Comment by minrk** [comment]
Cool, thanks!

**Comment by parente** [comment]
@ocefpaf @jakirkham When you get a chance, this one should be set for review / merge. Thanks!

File: recipes/sudospawner/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    - python
+    - jupyterhub >=0.4
+
+test:
+  imports:
+    - sudospawner
+
+about:
+  home: https://github.com/jupyterhub/sudospawner
+  license: BSD-3-Clause
```
Comment (ocefpaf) [inline recipes/sudospawner/meta.yaml]: @parente apparently people will scream now if we do not add the license file. Please see https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L42-L52 for an example.

Comment (parente) [inline recipes/sudospawner/meta.yaml]: Ah yes. Missed adding the file line. Should be there now.

**Comment by ocefpaf** [comment]
We already know that this is passing so let's get this merged. Thanks @parente!

**Comment by parente** [comment]
Thanks for the quick review @ocefpaf. 🍰

**Pull request merged**